scg-system/pom.xml

```diff
@@ -128,13 +128,12 @@
       <artifactId>logback-classic</artifactId>
     </dependency>
 
-    <!--
```

Comment:
Was this supposed to be removed or is this leftover from testing?

Reply:
Yeah, it's necessary. As part of pulling in the rdf-abac update (which has some increased logging - to make it easier to use as it logged NOTHING) it's needed. 

We might want to consider revisiting the logging (again) to try and consolidate it where we can.